spec/factories/route.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :route do
+    user
+    commitment
+    # start_date { Date.current }
```

Comment:
TODO right away: remove these comments.

Reply:
Actually I don't know if I care, they get fixed in the next commit so it might not be worth the effort at this stage.
If this were at work I might want to fix it but it depends who I was working with, TBH.